extensions/Worker.Extensions.ServiceBus/src/ServiceBusMessageActions.cs

```diff
@@ -62,6 +62,7 @@ internal ServiceBusMessageActions(Settlement.SettlementClient settlement)
         ///</remarks>
         protected ServiceBusMessageActions()
         {
+            _settlement = null!; // not expected to be used during mocking.
```

Comment:
Why was this needed?

Reply:
Nullability warning without it.

Reply:
It might be better to new up an instance if possible in case it does get used in mocking scenarios.

Reply:
I don't have any connection to new this up with. So, the failure would just happen later and be more obscure.